src/firmware/RC_firmware/RC_firmware.ino

```diff
@@ -0,0 +1,233 @@
+/* Firmware for Elsa
+   Test version
+   Author: Vincent Yuan, Modified: Nick Wu
+     Modified: James Asefa
+   Date: Oct 30, 2016
+*/
+
+#include <SoftwareSerial.h>
+#include <stdlib.h>
+#include <Servo.h>
+
+
+#define TRIM 8 // error margin for joysticks
+
+#define BAUD_RATE 115200
+#define BUFFER_SIZE 6
```

Comment:
Can you add a comment here explaining what `BUFFER_SIZE` is for? (Buffer for what?)